src/polar_multigrid.cpp

```diff
@@ -586,23 +591,23 @@ void gmgpolar::prepare_op_levels()
                 v_level[l]->define_nz_P();
 
                 // // Number of nonzeros in the matrix
-                if (gyro::icntl[Param::verbose] > 2)
+                if (gyro::icntl[Param::verbose] > 3)
                     std::cout << "nz_P: " << v_level[l]->nz_P << "\n";
                 v_level[l]->ri_prol = std::vector<int>(v_level[l]->nz_P);
                 v_level[l]->ci_prol = std::vector<int>(v_level[l]->nz_P);
                 v_level[l]->v_prol  = std::vector<double>(v_level[l]->nz_P);
                 v_level[l]->build_prolongation_bi();
 
                 // // Number of nonzeros in the matrix
-                if (gyro::icntl[Param::verbose] > 2)
+                if (gyro::icntl[Param::verbose] > 3)
                     std::cout << "nz_P_inj: " << v_level[l]->nz_P_inj << "\n";
                 v_level[l]->ri_prol_inj = std::vector<int>(v_level[l]->nz_P_inj);
                 v_level[l]->ci_prol_inj = std::vector<int>(v_level[l]->nz_P_inj);
                 v_level[l]->v_prol_inj  = std::vector<double>(v_level[l]->nz_P_inj);
                 v_level[l]->build_prolongation_inj();
 
                 // // Number of nonzeros in the matrix
-                if (gyro::icntl[Param::verbose] > 2)
+                if (gyro::icntl[Param::verbose] > 3)
                     std::cout << "nz_P_ex: " << v_level[l]->nz_P_ex << "\n";
```

Comment:
doesnt this fit better into "print out all kinds of variable values" ? so gyro::icntl[Param::verbose]>5